.template/addons/nginx/config/environments/production.rb

```diff
@@ -0,0 +1,11 @@
+gsub_file(
+  'config/environments/production.rb',
+  "config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?",
+  'config.public_file_server.enabled = false',
+)
+
+gsub_file(
```

Comment:
Do we really need this `gsub_file`? As if we decide to use nginx addon then `config.public_file_server.enabled = true` will not exists.

What do you think?

Reply:
it comes from the previous step @gutakk  https://github.com/nimblehq/rails-templates/pull/242

Here is the flow on generating the new rails project from template

```
# base config
https://github.com/nimblehq/rails-templates/pull/242

# addon
# nginx - optional...
```

Reply:
then should we only have this?

```ruby
gsub_file(
  'config/environments/production.rb',
  "config.public_file_server.enabled = true",
  'config.public_file_server.enabled = false',
)
```

IMO, if the the flow genere base config before nginx addon then  `config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?` will not exist as it is replaced by https://github.com/nimblehq/rails-templates/pull/242.

Please correct me if I'm wrong. I'm not 100% sure about this.

Reply:
The `config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?` will happend in case the user apply this ADDON to an existing project @gutakk 

We had [that option](https://github.com/nimblehq/rails-templates#use-the-template), the project isn't gen by our template so that might content the `config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
` which come from the Rails default one.

Reply:
I updated the PR description @gutakk, please have a look on it again.

Reply:
Thanks for explanation.